packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -54,13 +67,362 @@ import os = require('os');
 
 export interface LaunchRequestArguments
   extends DebugProtocol.LaunchRequestArguments {
+  /** comma separated list of trace selectors. Supported:
+	  * 'all': all
+	  * 'la': launch/attach
+	  * 'vh': variable handling
```

Comment:
va or vh? I see va being used below. Can we use an enum instead so we don't have to type these as strings and prevent typos like this.

Reply:
Change this to use full words.
la and vh do not make any sense to me.